service-binding/service/deployment/mssql-instance.yaml

```diff
@@ -11,4 +11,15 @@ spec:
     location: eastus
     resourceGroup: examples
     firewallStartIPAddress: "0.0.0.0"
-    firewallEndIPAddress: "255.255.255.255"
\ No newline at end of file
+    firewallEndIPAddress: "255.255.255.255"
+
```

Comment:
We need to wait for service instance to be ready here. Before creating service binding. AFAIK service binding tries 16 times for service instance to be ready and if its not ready in this time it would give up and we would not have any binding. @mszostok  Could you please confirm ?

Reply:
I see your point and changed the order. Not the instance gets created and verified, then binding, bindingusage and lamda are getting deployed in one chunk.